quantlib/instruments/payoffs.pyx

```diff
@@ -66,7 +67,7 @@ cdef class PlainVanillaPayoff(Payoff):
 
     def __init__(self, option_type, double strike, from_qlpayoff=False):
 
-        if isinstance(option_type, basestring):
+        if isinstance(option_type, str):     # Changed from basestring for Py2/3 compatibility
```

Comment:
Is it likely that we'll have Unicode option types under Python 2? If so, it might be advisable to wrap this in a separate routine.